mu-plugins/blocks/screenshot-preview/block.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Plugin name: Gutenberg: Pattern Previewer
```

Comment:
There's a bit of inconsistency between the naming and description here and in `block.json`. This file is quite particular about it being a for previewing patterns, whereas `block.json` seems more generic. Is that intentional?

Reply:
Nope, just missed this. Nice catch.